app-emulation/virt-manager: update dependencies

oz123 opened the pull request:
virt-manager has some issues fixed with python-libvirt 0.6.9
See for example: https://github.com/virt-manager/virt-manager/issues/203

Also, `pkgcheck scan` says :
```
 $ pkgcheck scan
...
  NonexistentBlocker: version 2.2.1-r3: nonexistent blocker DEPEND="!app-emulation/virtinst": no matches in repo history
 ...
```
Shall I remove this blocker too?


Package-Manager: Portage-3.0.9, Repoman-3.0.2
Signed-off-by: Oz Tiram <oz.tiram@gmail.com>

Dessa commented:
yes the blocker can be removed by now. it refers to an over 6 year old removed package: https://github.com/gentoo/gentoo-gitmig-20150809-draft/commit/4a1c03d781aed2c4cd1ce26fd9f0fcfb7e108cac

oz123 commented:
> yes the blocker can be removed by now. it refers to an over 6 year old removed package: [gentoo/gentoo-gitmig-20150809-draft@4a1c03d](https://github.com/gentoo/gentoo-gitmig-20150809-draft/commit/4a1c03d781aed2c4cd1ce26fd9f0fcfb7e108cac)

As requested, I removed the blocker and I also updated the dependencies of the git version (9999) ....

juippis commented:
fb6cc8062b097219740814d691f13cb3b7605e98

Dessa commented:
aw, i only now noticed a typo in the 9999 ebuild: libvirt-python-0.6.10 
-> libvirt-python-6.10.0

oz123 commented:
@Dessa  :+1:  my bad.
Do you want to fix it or shall I open another PR?

juippis commented:
I'll do it, thanks for reporting!
